meiki_ui/src/components/toolbar/Toolbar.svelte

```diff
@@ -4,7 +4,7 @@
 
 <nav
     class="flex flex-row w-full h-12 drop-shadow-card bg-slate-800 items-center px-4 py-7 gap-2"
-    transition:fly={{ y: -50, duration: 350, opacity: 1 }}
+    in:fly={{ y: -50, duration: 100, opacity: 1 }}
```

Comment:
Why does it matter? When do we have an out for the toolbar?

Reply:
We have an out when we transition to another page, so the user has to wait for the toolbar to out before it goes to the login page, which impacts UX, so changes it to only out

Reply:
Oh nice - great catch